docs/chain/collator.md

```diff
@@ -323,176 +297,175 @@ Secret phrase `very secure private key you should not use the example private ke
+
 :::info
 
 A session takes 600 blocks or around 120~ minutes
 
 :::
 
-## Add collator
+## Join the collator candidates pool
 
-:::info
+:::warning
 
-It is recommended to set the session keys before doing the following steps.
+These steps should be followed only once a collator node has successfully linked a session key to its address by following the steps above.
 
 :::
 
-The current number of collators is 17.
-
-Once the session keys has been set, the collator can join the pool of collators.
+The maximum number of **active** collators is currently 16 on Peregrine and 17 on Spiritnet.
 
-A collator staking amount:
+A collator staking amount must be:
 - minimum 10,000 KILT tokens 
 - maximum 200,000 KILT tokens.
 
-### Join collators
-
-The collator must make an extrinsic call from the `parachainStaking` pallet with the method `joinCandidates` with the desired stake entering the `candidatePool`.
+The collator must call an extrinsic from the `parachainStaking -> joinCandidates(stake)` with the desired stake to join the candidate pool.
 
 `Developer -> Extrinsics -> Submission`
 
-1. Using the selected account: Select the Collator Account 
-2. Submit the following extrinsic: parachainStaking -> joinCandidates(stake)
-3. Insert the staked KILT amount for the collator 
+1. Select the collator's KILT address as the extrinsic submitter (the *using the selected account* field)
+2. Submit the following extrinsic: `parachainStaking -> joinCandidates(stake)`
+3. Insert the staked KILT amount for the collator
+
+:::info
+
+A recent change in the blockchain metadata resulted in a change in the UI regarding how balances are shown. In the current version of PolkadotJS Apps, specifying 1 KILT requires adding 15 trailing `0`s. So, for instance, 1 KILT needs to be written as `1000000000000000`, while 10,000 KILT would be written as `10000000000000000000`.
+
+:::
 
 ![](/img/chain/parachainStaking-joinCandidates.png)
 
-A collator can check the current `SelectedCandidates` to see the position and required staked amount to join the network. 
+A collator can check the current top candidates to see the position and required staked amount to join the network. 
 
 `Developer -> Chain state -> Storage`
 
-1. Selected state query: `parachainStaking -> selectedCandidates(): Vec<AccountId>`
-2. Add the item with the "+" on the right side
+1. Selected state query: `parachainStaking -> topCandidates(): ParachainStakingSetOrderedSet`
+2. Execute the query by pressing the "+" button on the right side
 
-If the collator has enough self-stake and delegator stake they can be selected to collate.
-Once the collator has been chosen, the collator will be added to the `SelectedCandidate` pool.
+If the collator has enough self-stake and delegator stake it will be selected to collate.
+Once the collator has been chosen, it will be part of the `topCandidates()` set.
```

Comment:
```suggestion
A collator has been chosen, when he is part of the first 17 candidates in the `topCandidates()` set (16 for peregrine).
```

Reply:
I would refrain from putting actual numbers, as those need to be updated. Plus, we have mentioned few lines above that collators can check the minimum stake required to make in the active set, so I am not sure why we want to repeat that here.

Reply:
yes that's also fine. But the TopCandidates are candidates and only the first N candidates will become collators. It reads like everyone in the list is a collator

Reply:
I would then remove this line altogether, as I think it does not bring anything new on the table.